plugins/modules/omevv/omevv_firmware.py

```diff
@@ -0,0 +1,689 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Dell OpenManage Ansible Modules
+# Version 9.10.0
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: omevv_firmware
+short_description: Update the firmware of a specific host in the cluster
+version_added: "9.10.0"
+description: This module allows you to update the firmware of a specific host in the cluster.
+extends_documentation_fragment:
+  - dellemc.openmanage.omevv_auth_options
+options:
+  check_vSAN_health:
+    description:
+      - Whether to check vSAN health whilst update.
+      - C(true) checks the vSAN health whilst update.
+      - C(false) does not check the vSAN health whilst update.
+    type: bool
+  date_time:
+    description:
+      - Date and time when the job must run. This is applicable when I(run_now) is C(false).
+      - The format is YYYY-MM-DDThh:mm:ss<offset>.
+    type: str
+  delete_job_queue:
+    description:
+      - Whether to delete the job queue in iDRAC while performing firmware update.
```

Comment:
- 	Whether to delete the job queue in iDRAC while updating firmware.
- •	C(true) deletes the job queue in iDRAC while updating firmware.
- •	C(false) does not delete the job queue in iDRAC while updating firmware.

Reply:
Incorporated